libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -72,6 +73,7 @@ class elasticsearch_plugin_impl
       std::string bulk_line;
       std::string index_name;
       bool is_sync = false;
+      map <uint8_t , uint64_t > lookup_pow_table;
```

Comment:
Why not use a simple static const array with static initialization instead of a map? Would be much simpler IMO.
Also, use double values instead of uint64_t - they're always converted to double anyway, and it even saves you some casting when using the values.

Reply:
thanks, i agree and changed. cant make it `static` as a member of `elasticsearch_plugin_impl`. maybe somewhere else, however not sure where.